library/std/src/primitive_docs.rs

```diff
@@ -553,8 +553,10 @@ mod prim_pointer {}
 /// # Editions
 ///
 /// Prior to Rust 1.53, arrays did not implement `IntoIterator` by value, so the method call
-/// `array.into_iter()` auto-referenced into a slice iterator. That behavior is preserved in the
-/// 2015 and 2018 editions of Rust for compatability, ignoring `IntoIterator` by value.
+/// `array.into_iter()` auto-referenced into a slice iterator. Right now, the old behaviour
+/// is preserved in the 2015 and 2018 editions of Rust for compatability, ignoring
```

Comment:
```suggestion
/// is preserved in the 2015 and 2018 editions of Rust for compatibility, ignoring
```

Reply:
Oops, I had also misspelled that in <https://github.com/rust-lang/rust/pull/84147#pullrequestreview-641476700>.